src/openrct2/ride/RideRatings.h

```diff
@@ -54,11 +54,10 @@ struct RideRatingUpdateState
     uint16_t StationFlags;
 };
 
-static constexpr size_t RideRatingMaxUpdateStates = 4;
+static constexpr size_t RideRatingMaxUpdates = 4;
+using RideRatingUpdates = std::array<RideRatingUpdateState, RideRatingMaxUpdates>;
```

Comment:
Why this rename from `RideRatingUpdateStates` to `RideRatingUpdates`?

Reply:
I would also prefer to keep the name, its just plural of `RideRatingUpdateState` which makes most sense to me.

Reply:
When I move this into the `GameState_t` class then the type has the same name as the moved variable, which doesn't compile.

Perhaps an alternative to the rename would be to prefix the type with the namespace qualifiers, but I choose this way as it seems cleaner. 

Compilers also like to behave differently in such cases where type name and variable name are the same: https://godbolt.org/z/nWTsK6hz3

Reply:
You can use `::RideRatingUpdateStates RideRatingUpdateStates;`, this is an unfortunate thing due to our style convention.